feat: upgrade ioconnect-go to v0.0.5 and remove srv-did-vc for token signing

saitofun opened the pull request:
1. upgrade ioconnect-go to v0.0.5
2. remove `srv-did-vc` service dependency, sign and validate token use ioconnect-go directly.
3. fix unit test and e2e test
4. integrate `ioIDRegistry` contract for validate if clientID is registered.

saitofun committed:
feat: upgrade ioconnect-go to v0.0.5 and remove srv-did-vc for token signing

saitofun committed:
feat: update ioconnect-go library

saitofun committed:
fix: manager.ClientByClientDID

saitofun committed:
chore: add ioID-contracts abis and generate golang code

saitofun committed:
feat: add ecrypt response

saitofun committed:
feat: upgrade ioconnect-go to v0.0.8 and fix docker-compose

saitofun committed:
chore: add some log when fetched client did doc from contract

saitofun committed:
chore: parallel docker image build for commands

saitofun committed:
chore: add a mock did for debug

huangzhiran commented on cmd/e2etest/init.go:
```diff
@@ -35,23 +37,40 @@ var (
+	databaseDSN          string
+	coordinatorAddr      string
+	didAuthServerAddr    string
+	did                  bool
```
does this still used?

huangzhiran commented on clients/manager.go:
```diff
@@ -2,84 +2,108 @@ package clients
 
 import (
 	_ "embed" // embed mock clients configuration
```
does embed still used?

saitofun committed:
feat(clients): upgrade contracts and complete project permission

1. upgrade ioID contract, the default address is 0x06b3Fcda51e01EE96e8E8873F0302381c955Fddd
2.add
ProjectDevice contract, 0xF4d6282C5dDD474663eF9e70c927c0d4926d1CEb
3. complet default project
permission validation

saitofun committed:
chore: add encrypt response logs for debug

ququzone committed:
Fix smart contracts tests fail (#513)

saitofun committed:
docs: update docs (#511)

huangzhiran committed:
[scheduler] split project epoch offset (#512)

huangzhiran committed:
[task/dispatcher] dynamic set window size at every epoch (#514)

huangzhiran committed:
[smartcontracts/go] update contract abi (#515)

huangzhiran committed:
[persistence/contract] support IsValidProject contract api (#516)
